grunt/config/browserify.js

```diff
@@ -109,9 +109,46 @@ var addonsMin = {
   after: [minify, bannerify],
 };
 
+var reactDom = {
```

Comment:
TODO: these appeared to only build minified versions. Need to generate both regular and .min builds